CSS/inject.css

```diff
@@ -2,8 +2,12 @@
   --display-status: none;
 }
 
+body {
```

Comment:
Typically we want to avoid hardcoding fixed sizes if we can. Ideally we want to work on percent of display size or let our components fill available space.